### PR TITLE
Mark the language names as translation strings

### PR DESCRIPTION
**Pull request by folt**

### CODE REVIEW

File: {{cookiecutter.project_name}}/server/settings/components/common.py [hunk 1]
```diff
@@ -13,6 +13,7 @@
 from typing import Dict, List, Tuple, Union
 
 from server.settings.components import BASE_DIR, config
+from django.utils.translation import ugettext_lazy as _
```
Comment (DmytroLitvinov) [inline {{cookiecutter.project_name}}/server/settings/components/common.py]: Missed newline between import groups.

Comment (folt) [reply {{cookiecutter.project_name}}/server/settings/components/common.py]: I did everything right? 🙂

**Comment by sobolevn** [comment]
Do we need to `makemessages` after this change? Or these languages are already translated by django?

File: {{cookiecutter.project_name}}/server/settings/components/common.py (hunk 1 above)
Comment (DmytroLitvinov) [reply {{cookiecutter.project_name}}/server/settings/components/common.py]: Not exactly.
I added the correct code of importing ;)
Could you please change.

**Comment by sobolevn** [comment]
Why CI is green when `common.py` contains a style violation? 🤔

**Comment by DmytroLitvinov** [comment]
Yesterday, it was the same when I was working.

**Comment by sobolevn** [comment]
@DmytroLitvinov can you please dig into this problem? I am scared!

**Comment by DmytroLitvinov** [comment]
Answered on top of your preview message.

**Comment by sobolevn** [comment]
@folt can you please try to change `flake8 ./**/*.py` to `flake8 **/*.py` (without fixing style violations first)

I hope that CI will fail. And then the violation can be fixed. 
Sorry for the inconvenience! 😵

**Comment by DmytroLitvinov** [comment]
@sobolevn  Are we talking about template or project ready template?
Because you gave a link to code which used inside CI at GItlab(when the template is used for the project) and we are talking about this repo(wemake-django-template) which is using `travis.yml` file for triggeting CI at PR.
Or I am missing something and misunderstand you?

**Comment by DmytroLitvinov** [comment]
Very interesting ..
It is not working as next commands `flake8 **/*.py` or `flake8 ./**/*.py`.
But it raises error when use it like `flake8 .`
@sobolevn should we change to `flake8 .` maybe?

**Comment by DmytroLitvinov** [comment]
@sobolevn Check this [build](https://travis-ci.org/wemake-services/wemake-django-template/builds/521323977)

**Comment by DmytroLitvinov** [comment]
Let's continue to talk in separate issue: https://github.com/wemake-services/wemake-django-template/issues/745

**Comment by folt** [comment]
> Very interesting ..
> It is not working as next commands flake8 **/*.py or flake8 ./**/*.py.
> But it raises error when use it like flake8 .
> @sobolevn should we change to flake8 . maybe?

I observe the same result

**Comment by DmytroLitvinov** [comment]
Hi @sobolevn ,

Could we merge it and close?

**Pull request merged**